wiki/introduction/setting-up/en.md

```diff
@@ -207,4 +209,4 @@ Finally, reload the Gradle project by pressing this button in the Gradle menu:
 </picture>
 
 Once you're done with all of these steps, your mod is ready to be worked on!
-You can start by [Creating your First Item](../items/first-item)
+You can start by adding content via the [Creating your First Item](../items/first-item) and [Creating your First Block](../items/first-block) tutorials.
```

Comment:
yes, creating content via the "creating your first item" :yeefpineapple:

Reply:
it makes grammatical sense

Reply:
i'd probably add quotes tbh

`"Creating your First Item"` and `"Creating your First Block"`

Reply:
fine